### PR TITLE
Upgraded protobuf-java dependencies to 3.25.5

### PR DESCRIPTION
**Pull request by infvg**
## Description
Upgraded protobuf-java dependencies to version 3.25.5
## Motivation and Context
This upgrade was created to deal with CVEs found in lower versions
## Impact
None

## Release Notes
```
== RELEASE NOTES ==

General Changes
* Upgraded protobuf-java to version 3.25.5 :pr:`23797`
* Upgraded protobuf-java-util to version 3.25.5 :pr:`23797`
```

### CODE REVIEW

**Comment by agrawalreetika** [comment]
Should we upgrade it to some close to latest version may be `4.28.2` ?

File: pom.xml [hunk 1]
```diff
@@ -551,7 +552,13 @@
...
                 <groupId>com.google.protobuf</groupId>
                 <artifactId>protobuf-java</artifactId>
-                <version>3.25.1</version>
+                <version>${dep.protobuf-java.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>com.google.protobuf</groupId>
+                <artifactId>protobuf-java-util</artifactId>
+                <version>${dep.protobuf-java.version}</version>
```
Comment (agrawalreetika) [inline pom.xml:561]: Do we need to add this explicitly?

Comment (infvg) [reply pom.xml:561]: If we don't, 2.24.0 will be brought in by another dependency.

**Comment by infvg** [comment]
@agrawalreetika protobuf versions 3.x and 4.x have different APIs and upgrading the major version might result in issues.

https://protobuf.dev/support/version-support/
The 3.25.x version will continue to receive support until 31 March 2026.

**Review by agrawalreetika — APPROVED** [review]

**Comment by agrawalreetika** [comment]
Looks like there are some dependency issues in `presto-bigquery` connector, please check and fix those

**Review by tdcmeehan — APPROVED** [review]

**Pull request merged**